ui/components/MesheryPatterns.js

```diff
@@ -508,11 +519,13 @@ function MesheryPatterns({
   }
 
   const handlePublishModal = (ev, pattern) => {
-    ev.stopPropagation();
-    setPublishModal({
-      open : true,
-      pattern : pattern
-    });
+    if (!isLocalUser) {
```

Comment:
Can this additional check be avoided?

Reply:
@harkiratsm Probably! Although I don't want any loose ends when dealing with modals which are controlled from various components.

Reply:
@harkiratsm added latest changes. Please review.